lib/sentry/envelope.ex

```diff
@@ -34,6 +34,41 @@ defmodule Sentry.Envelope do
     }
   end
 
+  @doc """
+  Creates a new envelope containing the client report.
+  since: "10.8.0"
```

Comment:
`@doc since` is an ExDoc feature. I literally meant:

```elixir
@doc since: "10.8.0"
```

after the `@doc` string. There are tons of examples in this repo 🙃